### PR TITLE
Update quickstart doc 8.0

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   docs/quickstart-overview.md

### CODE REVIEW

**Review by patrickbirch — APPROVED** [review]
good

File: docs/quickstart-overview.md [hunk 1]
```diff
@@ -16,6 +16,16 @@ You can install Percona XtraBackup using different methods:
 
 * [Use Docker](docker.md)
 
+## For superior and optimized performance
+
+[Percona Server for MySQL](https://docs.percona.com/percona-server/8.0/) is a freely available, fully compatible, enhanced, and open source drop-in replacement for any MySQL database. It provides superior and optimized performance, greater scalability and availability, enhanced backups, increased visibility, and instrumentation. Percona Server for MySQL is trusted by thousands of enterprises to provide better performance and concurrency for their most demanding workloads.
+
```
Comment (patrickbirch) [inline docs/quickstart-overview.md:24]: add a link to the PS install doc

File: docs/quickstart-overview.md [hunk 2]
```diff
@@ -16,6 +16,16 @@ You can install Percona XtraBackup using different methods:
...
 ## For high availability
 
-[Percona XtraDB Cluster (PXC)](https://docs.percona.com/percona-xtradb-cluster/8.0/) is a 100% open source, enterprise-grade, highly available clustering solution for MySQL multi-master setups based on Galera. PXC helps enterprises minimize unexpected downtime and data loss, reduce costs, and improve performance and scalability of your database environments supporting your critical business applications in the most demanding public, private, and hybrid cloud environments. 
\ No newline at end of file
+[Percona XtraDB Cluster (PXC)](https://docs.percona.com/percona-xtradb-cluster/8.0/) is a 100% open source, enterprise-grade, highly available clustering solution for MySQL multi-master setups based on Galera. PXC helps enterprises minimize unexpected downtime and data loss, reduce costs, and improve performance and scalability of your database environments supporting your critical business applications in the most demanding public, private, and hybrid cloud environments. 
+
```
Comment (patrickbirch) [inline docs/quickstart-overview.md:30]: add a link to the pxc install doc

File: docs/quickstart-overview.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-overview.md:24]: done

File: docs/quickstart-overview.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-overview.md:30]: done

**Pull request merged**